Automattic/pocket-casts-ios
# Reimagine Sharing: Fix trim handle lag on long episodes

| 📘 Part of: #1910 |
|:---:|

Fixes #2124

There are 2 primary performance changes here:
* Use the newer `formatted` APIs where we can to gain the cached formatter
* Switch to `LazyHStack` for the invisible tick marks - this was the primary performance fix

## To test

* Play an episode with a long length (4+ hours) - I chose an 8 hour episode with Elon Musk on the Lex Fridman podcast
* Move the playback position somewhere in the middle of the episode (this is to check that programmatic scrolling still works)
* Tap "Share"
* Tap the "Clip" option
* Verify that trim handles are responsive and not laggy
* Verify that moving the handles still updates the timestamps along the bottom (the formatter change)
* Verify that clipping still works correctly

## Checklist
